### PR TITLE
ACMS-1826: Add Acquia global commands in DRS.

### PR DESCRIPTION
**Pull request by rajeshreeputra**
**Motivation**
Fixes #[ACMS-1826](https://acquia.atlassian.net/browse/ACMS-1826)

**Proposed changes**

**Alternatives considered**

**Testing steps**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -34,6 +35,12 @@
         "phpro/grumphp-shim": "^1.5",
         "phpunit/phpunit": "^9 || ^10"
     },
+    "repositories": {
+        "acquia_global_commands": {
+            "type": "vcs",
+            "url": "git@github.com:vishalkhode1/acquia_global_commands.git"
+        }
+    },
```
Comment (rajeshreeputra) [inline composer.json:43]: @vishalkhode1 could we initiate the migration of the 'acquia_global_commands' repository from 'vishalkhode1' to the 'acquia' organization? Additionally, release the development version on packagist, so no need to add this section in composer.json file.

**Review by chandan-singh7929 — APPROVED** [review]
LGTM

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,5 @@
 vendor
+.idea
```
Comment (apathak18) [inline .gitignore:2]: @rajeshreeputra line# 2 & 3 isn't same? If we only add .idea then I guess it would work

Comment (chandan-singh7929) [reply .gitignore:2]: We should keep `.idea/` one

Comment (rajeshreeputra) [reply .gitignore]: Updated the same to keep .idea/

**Review by apathak18 — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (vishalkhode1) [reply composer.json:43]: Yes, will do that once we all (i.e Acquia CMS team, blt team etc.) are align with this approach.

**Pull request merged**